internal/ingress/controller/parser/parser.go

```diff
@@ -1113,9 +1113,9 @@ func (p *Parser) getKongIngressFromIngress(ing *networking.Ingress) (
 
 	ki, err := p.store.GetKongIngress(ing.Namespace, ing.Name)
 	if err == nil {
-		return ki, err
+		return ki, nil
 	}
-	return nil, nil
+	return nil, err
```

Comment:
The reason we silently ignore this is because we don't want to log a 404 error here. This branch is hit almost always for most deployments.

Reply:
Understand, I have reverted this change